src/main/kotlin/io/github/intellij/dlanguage/features/DCommenter.kt

```diff
@@ -0,0 +1,59 @@
+package io.github.intellij.dlanguage.features
+
+import com.intellij.lang.CodeDocumentationAwareCommenter
+import com.intellij.psi.PsiComment
+import com.intellij.psi.tree.IElementType
+import io.github.intellij.dlanguage.psi.DlangTypes
+
+class DCommenter : CodeDocumentationAwareCommenter {
+    override fun getLineCommentPrefix(): String {
+        return "//"
+    }
+
+    override fun getBlockCommentPrefix(): String {
+        return "/*"
+    }
+
+    override fun getBlockCommentSuffix(): String {
+        return "*/"
+    }
+
+    override fun getCommentedBlockCommentPrefix(): String? {
+        return null
+    }
```

Comment:
a convenience with Kotlin is that these getters can be shortened to a single line. eg:

```
override fun getBlockCommentSuffix(): String = "*/"
```